src/monitoring/request_timer.rs

```diff
@@ -0,0 +1,28 @@
+use rocket::fairing::{Fairing, Info, Kind};
+use chrono::Utc;
+use rocket::{Request, Response, Data};
+
+pub struct RequestTimer();
+
+impl Fairing for RequestTimer {
+    fn info(&self) -> Info {
+        Info {
+            name: "RequestTimer",
+            kind: Kind::Request | Kind::Response,
+        }
+    }
+
+    fn on_request(&self, request: &mut Request, _data: &Data) {
+        let instant = Utc::now();
+        log::info!("on_request: {}", &instant.to_rfc3339());
```

Comment:
sorry I was not clear here ... I would remove the `on_request` and `on_response` for the final version (or make them `debug`) and only log the final information to `info`.